src/config/test-config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   awsAccountId: '000000000000',
   awsRegion: 'eu-west-1',
   corsOriginUrl: 'https://scp.mockDomainName.com',
-  domainName: 'scp.mockDomainName.com',
+  domainName: 'scp.biomage.net',
```

Comment:
why change this? and why isn't the cors one changed too?

Reply:
Because I check the `domainName` in my new changes, so I could use it in my tests directly if it was the production one

Reply:
I didn't change the cors becuase I didn't need it, would you prefer me to change it too?

Reply:
yeah I think so